### ISSUE TITLE
nil net_packet_ipv6 events

### ISSUE THREAD

**Issue by OriGlassman** [aquasecurity/tracee#2687]
## Description

./dist/tracee-ebpf -t c -c pcap:container -c pcap:process -t event=net_packet_ipv*,net_packet_icmp*

## Output of `tracee -v`:

```
Tracee version "v0.11.0"
```

## Output of `uname -a`:

```
Linux ip-172-31-28-94 5.15.0-1026-aws #30~20.04.2-Ubuntu SMP Fri Nov 25 14:53:22 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

## Additional details 

![MicrosoftTeams-image](https://user-images.githubusercontent.com/39296766/217244501-b50cd4c6-43fb-4d2d-a431-f51fa795a75e.jpg)

**Comment by yanivagman** [comment]
@rafaeldtinoco shouldn't #2685 have closed this one?

**Comment by rafaeldtinoco** [comment]
@OriGlassman did you test fix provided by that PR ?

**Comment by rafaeldtinoco** [comment]
Probably fixed by: https://github.com/aquasecurity/tracee/pull/2685

**Comment by yanivagman** [comment]
Closing this as fixed by #2685
